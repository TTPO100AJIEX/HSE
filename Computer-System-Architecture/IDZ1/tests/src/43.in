100
-137 705 -392 -409 -172 -549 -631 234 119 516 -21 -556 667 353 -379 502 703 292 421 274 -152 -411 -330 -632 -745 -653 75 -598 -272 188 -332 -564 -682 -138 -390 -81 228 -728 -5 -119 -108 169 758 147 -641 -166 -445 -467 -588 750 -740 -235 390 -141 257 -670 166 -467 100 -473 -135 341 34 761 -458 178 518 587 744 571 644 509 567 -16 604 -422 -50 428 -680 -744 -17 479 202 -203 515 515 -626 -131 688 37 267 477 -679 680 -504 483 607 671 270 354