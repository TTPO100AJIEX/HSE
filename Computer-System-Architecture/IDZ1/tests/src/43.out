-137 705 -409 -172 -631 234 119 516 -556 667 -379 502 703 292 421 -411 -330 -745 -653 75 -598 -272 188 -682 -138 -390 -81 228 -728 -5 -119 -108 169 758 -641 -166 -588 750 -740 -235 390 -141 257 -670 166 -467 100 -473 -135 341 34 761 -458 178 518 587 744 571 644 509 567 -16 604 -422 -50 428 -744 -17 479 -203 515 515 -626 -131 688 37 267 477 -679 680 -504 483 607 671 270 354 